docs/guides/Mobx-Integration.md

```diff
@@ -0,0 +1,76 @@
+# Mobx Integration
+
+Handling your app's navigation state in mobx, is similar to redux. You can pass your own navigation prop to a navigator. Your navigation prop must provide the current state, as well as access to a dispatcher to handle navigation options.
+
+With mobx, your app's state is defined by observable attributes. But react-navigation expects a dispatcher function, to trigger the state changes on our nav state observable:
```

Comment:
nit: the Mobx official documentation (https://mobx.js.org/#observable-state) refers to these "observable attributes" as "observable properties"